IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinUtilBase.m

```diff
@@ -492,4 +500,52 @@ + (MSIDWPJMetadata *)readWPJMetadataWithSharedAccessGroup:(NSString *)sharedAcce
     }
     return nil;
 }
+
++ (SecKeyRef)getSessionTransportKeyFromSecureEnclaveForTenantId:(NSString *)tenantId context:(id<MSIDRequestContext>)context
```

Comment:
I suggest making the function name: `getSessionTransportKeyRefFromSecureEnclaveForTenantId` because there is no way to read the private key

Reply:
Addressed